mleap-core/src/main/scala/ml/combust/mleap/core/feature/HashingTermFrequencyModel.scala

```diff
@@ -24,11 +24,17 @@ object HashingTermFrequencyModel {
       case d: Double => hashLong(java.lang.Double.doubleToLongBits(d), seed)
       case s: String =>
         val utf8 = s.getBytes("UTF-8")
-        hashUnsafeBytes(utf8, Platform.BYTE_ARRAY_OFFSET, utf8.length, seed)
+        if (version == VERSION_1) {
+          hashUnsafeBytes(utf8, Platform.BYTE_ARRAY_OFFSET, utf8.length, seed)
+        } else {
+          hashUnsafeBytes2(utf8, Platform.BYTE_ARRAY_OFFSET, utf8.length, seed)
+        }
       case _ => throw new IllegalStateException("HashingTF with murmur3 algorithm does not " +
         s"support type ${term.getClass.getCanonicalName} of input data.")
     }
   }
+  val VERSION_1: Int = 1
+  val VERSION_2: Int = 2
```

Comment:
maybe rename this to `HASH_UNSAFE_BYTES_VERSION_1` and `HASH_UNSAFE_BYTES_VERSION_2` to give a bit more clarity what the version applies to?